.github/ISSUE_TEMPLATE/general-question.yml

```diff
@@ -27,27 +24,12 @@ body:
       description: >-
         Please post any supporting code around your question here.
 
-  - type: textarea
-    attributes:
-      label: Other information
-      description: >-
-        Please provide any attempts on answering the question, or any related information you already found here.
-
   - type: dropdown
     attributes:
       label: Which package?
       options:
         - vizro
         - vizro-ai
-  - type: input
-    attributes:
-      label: Package version
-  - type: input
-    attributes:
-      label: Python version
```

Comment:
Do we ask for the package and python elsewhere? I think this might be useful to keep?

Reply:
I don't think it's relevant for a general question, just for bug reports (and even then we don't really care about Python version).